deploy.yaml

```diff
@@ -317,3 +318,114 @@
       command: oc --namespace "{{ orchestrator_namespace }}"
                   start-build osbs-buildroot
       changed_when: true
+
+
+- name: Run cachito containers on openshift
+  hosts: osbs_box_host
+  gather_facts: false
+  vars:
+    # This is a constant, please do not override
+    cachito_components:
+      - cachito-volumes
+      - cachito-api
+      - cachito-worker
+      - cachito-rabbitmq
+      - cachito-athens
+  tags:
+    - cachito
```

Comment:
`cachito` tag should also be added to `Prepare for deployment of openshift applications`